apache/superset
# refactor: icon to icons for basicerror componenet

### SUMMARY
This pr refactors the basicerror icon to icons.

### BEFORE/AFTER SCREENSHOTS OR ANIMATED GIF

before 
<img width="271" alt="Cannot load filter" src="https://user-images.githubusercontent.com/17326228/123161138-e708f580-d423-11eb-8bf2-aa269ce2d293.png">
<img width="470" alt="Screen Shot 2021-06-23 at 10 53 33 AM" src="https://user-images.githubusercontent.com/17326228/123161494-4e26aa00-d424-11eb-9f72-64874c7428a0.png">
after
<img width="271" alt="Cannot load filter" src="https://user-images.githubusercontent.com/17326228/123161163-ee300380-d423-11eb-8d94-dd8b52c1be62.png">
<img width="470" alt="Screen Shot 2021-06-23 at 10 53 33 AM" src="https://user-images.githubusercontent.com/17326228/123161494-4e26aa00-d424-11eb-9f72-64874c7428a0.png">

### TESTING INSTRUCTIONS
To test you must enable nativefilters feature flags and check the filterconfigmodal and filterbox for the basic error alert.

### ADDITIONAL INFORMATION
